scripts/pre_ingest_sanitisation.py

```diff
@@ -0,0 +1,236 @@
+# For an annotated version, use the ipython notebook.
```

Comment:
It's not, I made this myself because I thought that having it only exist as an IPython notebook might make people sad, so I made this for convenience, but I take your point about only having one. Given that I find having it as a notebook very useful for future poking I'm going to just delete the script.